notifier/notifier_test.go

```diff
@@ -443,13 +411,10 @@ func (a alertmanagerMock) url() *url.URL {
 func TestLabelSetNotReused(t *testing.T) {
 	tg := makeInputTargetGroup()
 	_, _, err := alertmanagerFromGroup(tg, &config.AlertmanagerConfig{})
-	if err != nil {
-		t.Fatal(err)
-	}
 
-	if !reflect.DeepEqual(tg, makeInputTargetGroup()) {
-		t.Fatal("Target modified during alertmanager extraction")
-	}
+	testutil.Ok(t, err)
+
+	testutil.Assert(t, reflect.DeepEqual(tg, makeInputTargetGroup()), "Target modified during alertmanager extraction")
```

Comment:
Can you not use testutil.Equals here?

Reply:
I didn't use `testutil.Equals` because it can not print custom message and I want respect original fatal message.

Reply:
That doesn't give us the diff then. I'd suggest making the message a comment.

Reply:
Ok, I'll update it to `testutil.Equals`. Thank for feedback